pullrequest.go

```diff
@@ -84,18 +84,22 @@ type pullRequestQuery struct {
 }
 
 func (q pullRequestQuery) Filter() string {
+	filterStr := ""
 	switch {
 	case q.org != "":
-		return "org:" + q.org
+		filterStr += " org:" + q.org
+		fallthrough
```

Comment:
this might be a matter of taste, but IMO better to avoid `fallthrough` and create a specific case for when you want to do multiple things

Reply:
using `fallthrough` in case statements makes refactoring them later a bit extra spooky IMO :ghost:

Reply:
Agree, and I'm happy to do away with the `switch` statement altogether, I just recall that keeping `switch` statements instead of multiple `if` statements has been requested by Go linters in the past.

Reply:
oh yeah, I know which linter you mean - if we can avoid `fallthough` it would be great!

Reply:
Got it, changed the code to use `if` statements instead.